### DISCUSSION TITLE
升级2.11.3之后拉库提示网络异常

### DISCUSSION THREAD

**Discussion by qilongcainiao** [whyour/qinglong#1231] [Q&A]
## 开始执行... 2022-03-17 09:58:40

开始克隆仓库 https://ghproxy.com/github.com/KingRan/JDJB.git 到 /ql/repo/KingRan_JDJB

更新/ql/repo/KingRan_JDJB失败，请检查网络...

## 执行结束... 2022-03-17 09:58:40  耗时 0 秒

**Comment by cc9014** [comment]
仓库地址不对，需要去掉ghproxy.com这个，外国主机直接拉，国内主机自备或者网上找其他的加速地址。
